modules/users/manifests/benthorner.pp

```diff
@@ -1,11 +1,11 @@
 # Create the benthorner user
 class users::benthorner {
   govuk_user { 'benthorner':
-    ensure   => absent,
+    ensure   => present,
```

Comment:
Oh, oops, I just removed the `ensure` line entirely in #9942. Will that not work?

Reply:
Hmm...not sure. My public key has changed in any case...